cmd/katib-controller/v1alpha3/main.go

```diff
@@ -83,7 +85,7 @@ func main() {
 	}
 
 	log.Info("Setting up webhooks")
-	if err := webhook.AddToManager(mgr); err != nil {
+	if err := webhook.AddToManager(mgr, int32(webhookPort)); err != nil {
```

Comment:
why not `var webhookPort int32` in line 42?

Reply:
Because defining the variable as `int32` fails in `flag.IntVar` with 

```
v1alpha3/main.go:47:14: cannot use &webhookPort (type *int32) as type *int in argument to flag.IntVar
```

So I tried `flag.Var`, but `int32` does not have `flag.Set` method implemented

```
*int32 does not implement flag.Value (missing Set method)
```

So the simplest solution seemed to be to just load `int` and change the type. Do you have different suggestion for solution?